Autopilot/Path Manager/MainBatterySensor.h

```diff
@@ -11,14 +11,15 @@
 #ifdef	__cplusplus
 extern "C" {
 #endif
+    
+#define CELLS 2
```

Comment:
This should be renamed to something more specific, such as MOTOR_BATTERY_CELL_COUNT or whatever